### ISSUE TITLE
Separate LCE classes into a separate module

### ISSUE THREAD

**Issue by dimsuz** [sockeqwe/mosby-conductor#24]
Currently when using `mvp-conductor` an LCE classes automatically add dependencies to mosby's LCE and ViewState modules:

```
+--- com.hannesdorfmann.mosby3:mvp-conductor:3.0.0@aar
|    \--- com.hannesdorfmann.mosby3:mvp-lce:3.0.0@aar
|         \--- com.hannesdorfmann.mosby3:viewstate:3.0.0@aar
```

In my projects I do not use those modules, because I provide my own LCE implementations. But currently I still have these two additional dependencies which I do not even use...

It would be great to have `mvp-conductor-lce` or something like this, so that there can be more fine grained tuning.

Are you planning on doing something like this or do you oppose these changes? 
I might try to come up with PR if this is acceptable and if you didn't already start this.

**Comment by sockeqwe** [comment]
Yes, that is something I have considered to do for `3.0` but I have dropped the idea for `3.0`.
The reason was that it would require Mosby  (not the conductor repository) to extract the LCE View Interface into it's own module and publish it as it's own jar so that both `com.hannesdorfmann.mosby3:mvp-lce:x.y.z` and `com.hannesdorfmann.mosby3:conductor-lce:x.y.z` can include it.

Given the fact that `mosby-conductor` is not very popular, I found it not worthwhile to add this separate module just for the LCE view interface.

**Comment by dimsuz** [comment]
Ah, I see. Those modules, sometimes they seem to multiply at an unusually fast rate! :)

This issue should be closed then I guess...
